### ISSUE TITLE
SecurityError: Failed to read the 'localStorage' property from 'Window'

### ISSUE THREAD

**Issue by dennisroche** [cribeiro84/azure-devops-pull-request-hub#136]
```
There was an error during the extension load: SecurityError: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
```

Occurs in:

- Chome 84.0.4147.89 - displays the above error.
- Edge Chromium 84.0.522.44 - it shows for a moment and then disappears. 

Works in:
- Firefox Developer
- Edge Trident (before Chromium)

Stack trace

```
ms.vss-web.platform-content.es6.T1WfB1Bq6sDuC5Cr.min.js:1 No permissions found for security namespace '2e9eb7ed-3c0a-47d4-87c1-0ffdd275fd87' and token 'repoV2/19aad2a7-09fa-47b2-b0da-79e5f92574fa/3d7feb97-940c-43b7-9477-516b1dd7eeb3/'. Ensure that the effective permissions are included in shared data.

Uncaught (in promise) DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
    at e.load (https://caribeiro84.gallerycdn.vsassets.io/extensions/caribeiro84/pull-request-manager-hub/1.20200728.1/1595985637662/build/static/js/main.4f80e67b.chunk.js:1:825)
```

**Comment by dennisroche** [comment]
A quick look at the last PR merged, seeing references to `localStorage`. 

https://github.com/cribeiro84/azure-devops-pull-request-hub/blob/debeab23e73e42a69e8c72b9621be48a878a063d/src/common.tsx#L18

**Comment by cribeiro84** [comment]
Thanks @dennisroche for reporting the issue and I'm sorry for this inconvenience. We use the localStorage to control some features (save filters, PR last visit). Do you know if localStorage is blocked on your browser? Is there any restriction from your side?
In meantime, we'll work on a fix to ignore the features in case localStorage is not available.

**Comment by dennisroche** [comment]
@cribeiro84 no changes to browser settings between yesterday (working) and today (not working). I tested the extension on 2 laptops with the different browsers and asked my team to also verify.

Does the extension manifest need to specify use of local storage? 

A quick read of the docs indicates that `localStorage` isn't available in extensions and that any preferences should be persisted via [Data Storage](https://docs.microsoft.com/en-us/azure/devops/extend/develop/data-storage?view=azure-devops)

**Comment by cribeiro84** [comment]
We will work on this fix and I hope to get a new release for tomorrow. Sorry for this inconvenience.

**Comment by dennisroche** [comment]
Confirmed fixed in #139. 

I needed to clear local state before it worked, otherwise it got an exception in `DropdownFilterBarItem` at `Line 87`

```
_this.renderSelectedItems = function (selection, items) {
            var selectedText = items[selection.value[0].beginIndex].text || "";
```

With `items[selection.value[0].beginIndex]` being `null`.

**Comment by dennisroche** [comment]
**UPDATE**. Access to `localStorage` is denied if `3rd party cookies` are disabled. As the content for the extensions are served from a CDN this breaks unexpectedly.

Fix and suggest changing the error message is to add `https://[*.]vsassets.io` to the allowed list in the browser (e.g. chrome://settings/cookies)
